modules/memoizable.rb

```diff
@@ -0,0 +1,25 @@
+module Memoizable
```

Comment:
Thank you for this module Sienna! It was amazing to learn of these shortcuts that also cut down our run time. I really liked them and learned a lot.